build/webpack.base.config.js

```diff
@@ -39,7 +39,6 @@ function getCssLoaders(config) {
 
     if (config.type === 'server') {
         if (config.extractCss) {
-            cssLoaders[0].options.onlyLocals = true;
```

Comment:
Per https://github.com/webpack-contrib/css-loader#exportonlylocals, this seems to be only relevant for CSS Module usage now - so removing